docs/docs/guides/labs/dg/creating-a-project.md

```diff
@@ -10,32 +10,32 @@ import InstallUv from '@site/docs/partials/\_InstallUv.md';
 
 <DgComponentsPreview />
 
-`dg` allows you to create a special type of Python package, called a _project_, that defines a [Dagster code location](/deployment/code-locations/managing-code-locations-with-definitions).
+The `create-dagster` CLI allows you to create a special type of Python package, called a _project_, that defines a [Dagster code location](/deployment/code-locations/managing-code-locations-with-definitions).
 
 :::note
 
-Dagster projects created with `dg` are compatible with Dagster Components. For more information, see the [Dagster Components documentation](/guides/labs/components).
+Dagster projects created with `create-dagster` are compatible with Dagster Components. For more information, see the [Dagster Components documentation](/guides/labs/components).
 
 :::
 
 ## Prerequisites
 
-Before creating a project with `dg`, you must [install `dg`](/guides/labs/dg#installation).
+Before creating a project, you must [install `create-dagster`](/guides/labs/dg#installation). If you're using `uv`, you can run `create-dagster` using `uvx`, without needing to install it first.
```

Comment:
I can leave a more thorough review later tonight, but quick note to fix the build -- the link here should be `/guides/labs/dg#installing-the-create-dagster-cli`